### PR TITLE
nHCal fix rotated segmentation

### PR DESCRIPTION
**Pull request by lkosarz**
### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [x] Other: Fix rotated segmentation for backward HCal

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

Tests have been performed

### CODE REVIEW

File: configurations/hcal_only.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+features:
```
Comment (veprbl) [inline configurations/hcal_only.yml:1]: Rename to backward_hcal_only.yml

Comment (veprbl) [reply configurations/hcal_only.yml:1]: or nhcal_only.yml

**Review by lkosarz — COMMENTED** [review]
Done

**Review by lkosarz — COMMENTED** [review]
Done

**Review by veprbl — APPROVED** [review]

**Comment by veprbl** [comment]
Can you also address failing overlap checks: https://github.com/eic/epic/actions/runs/9601322569/job/26479729341?pr=748

**Comment by lkosarz** [comment]
I did test these overlaps and somehow couldn't believe they are real. Now I fixed them

**Review by veprbl — APPROVED** [review]

**Pull request merged**